app/server/utils/ussd/token_processor.py

```diff
@@ -198,7 +198,7 @@ def fetch_exchange_rate(user: User):
 
         exchange_limit = rounded_dollars(TokenProcessor.get_default_exchange_limit(default_limit, user))
         exchange_rate = round_to_sig_figs(exchange_rate_full_precision, 3)
-        exchange_sample_value = rounded_dollars(exchange_rate_full_precision * float(1000))
```

Comment:
for documentation purposes: the rounding function that rounds money to dollars is meant to take an input thats in cents (for example: rounding a balance of 5000 cents into 50 dollars)

Reply:
done